### DISCUSSION TITLE
How do I regain my account? I lost my computer and changed my phone, I never setup 2FA because I assumed forgot Password just sent an email. So what do I do, is there anyone at github I can contact?

### DISCUSSION THREAD

**Discussion by ImproverMan** [community/community#135105] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

There is literally nothing I can do. When I lost my password word document with my password, I lost all access to my account. My company email is attached to this account and I can't unlink it either. So what exactly are my options now?

**Comment by ahmedembeddedxx** [comment]
There is nothing the community can do, the best you can do is either try to retrieve the security code documents. If that doesn't work, you can create a new account and ask github support to grant you access to the account. However, to be completely honest, I've been trying to get a response to my ticket for the past five weeks, it is still open. You have the option to request that your account be canceled by your company. It's preferable to nothing.

**Reply by ImproverMan** [reply]
I am so mad right now that this is the way it is. Why can't I just get a reset on email? I am fuming right now that it is this way and that github did not account for the fact that the 2FA in reality is relatively new. 2023, and the fact that it is SUCH A HUGE part of the password reset, I don't remember ever getting an email about the new requirements.

**Reply by ImproverMan** [reply]
if anything, they could atleast Unlink my email as my secondary email, not my primary is forever lost now and I can't use it again

**Reply by ahmedembeddedxx** [reply]
You can try submitting a ticket for your alternative account, it IS quite frustrating since i lost my account recently and had to start everything from scratch because they think that my account is possibly suspicious and has been flagged.

**Reply by ImproverMan** [reply]
I have a quick question. If an email has been verified, but it does not have an account, can I not create an account on that email?
I beg to gods you say yes

**Comment by ebndev** [comment]
Hi @ImproverMan,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
